recipes/python/retail/retail.config.json

```diff
@@ -29,7 +44,11 @@
       {
         "key": "ACP_DSW_TRAINING_XDM_SCHEMA",
         "value": "/_customer/default/DSWRetailSales"
```

Comment:
Curious - is this still used?

Reply:
Yes, we needed these values for recent evaluator changes. So currently R/Python and Scala are using these constants.